ENIGMAsystem/SHELL/Platforms/General/POSIX/main.cpp

```diff
@@ -25,9 +25,9 @@ void initialize_directory_globals() {
   char const *env = getenv("TMPDIR");
 
   if (env == 0)
-    env = "/tmp/";
+    env = "/tmp";
 
-  enigma_user::temp_directory = env; 
+  enigma_user::temp_directory = env + string("/");
```

Comment:
The TMPDIR environment variable may or may not contain a trailing slash, and this is less clear on the `env` constant. It's better to handle the former conditionally and the latter using a return statement.

Reply:
@JoshDreamland Agreed. I'll take care of it.

Reply:
Resolved, unless you see somewhere I messed up.